pkg/client/unversioned/clientcmd/api/types.go

```diff
@@ -62,6 +62,8 @@ type Cluster struct {
 	LocationOfOrigin string
 	// Server is the address of the kubernetes cluster (https://hostname:port).
 	Server string `json:"server"`
+	// Servers is an array of addresses to the kubernetes apiservers with the format: (https://hostname:port)
+	Servers []string `json:"servers,omitempty"`
```

Comment:
the names here are unclear if the host specified in `Server` should also be specified in `Servers`. Is the goal to make people specify a primary server in `Server` and fallbacks in `Servers`, or to specify the whole list in `Servers` and leave `Server` empty?